File: src/com/young/simpledict/service/event/BaseEvent.java
package com.young.simpledict.service.event;

/**
 * Author: taylorcyang
 * Date:   2014-10-21
 * Time:   17:40
 * Life with passion. Code with creativity!
 */
public class BaseEvent {
    Object eventCode;
    public BaseEvent setEventCode(Object code) {
        eventCode = code;
        return this;
    }
}

File: app/src/main/java/com/young/simpledict/service/event/BaseEvent.kt
package com.young.simpledict.service.event

/**
 * Author: landerlyang
 * Date:   2014-10-21
 * Time:   17:40
 * Life with passion. Code with creativity!
 */
open class BaseEvent(var eventCode: Any) {

    fun setEventCode(code: Any): BaseEvent {
        this.eventCode = code
        return this
    }

    companion object {
        const val EMPTY_EVENT_CODE = -1
    }
}
